test/controllers/WebchatControllerSpec.scala

```diff
@@ -133,12 +133,12 @@ class WebchatControllerSpec
       redirectLocation(result) mustBe Some(routes.WebchatController.serviceUnavailable.url)
     }
 
-    "render online services helpdesk page" in {
```

Comment:
You need to create two tests. One for when the flag showSACUI is true and one for when the flag is false

Reply:
Hi Tony thanks for the review and i have added the test as per your recommendations